participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserManagementProfileService.java

```diff
@@ -37,7 +37,7 @@ public String deactivateAccount(
       String userId, DeactivateAcctBean deactivateBean, AuditLogEventRequest auditRequest);
 
   public EmailResponse resendConfirmationthroughEmail(
-      String applicationId, String securityToken, String emailId);
+      String applicationId, String securityToken, String emailId, String appName);
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Fixed review comment.